src/policy/protectpayload.rs

```diff
@@ -0,0 +1,104 @@
+//! The default policy module, which enforces no policy.
```

Comment:
Change the description here to explain what this policy does.

In general the rust module-level documentation is formatted as a title, followed by a blank line, and a descriptive paragraph. Like this:

```rs
//! Example of Module-Level Documentation
//!
//! The module-level documentation start with `//!` instead of `//!`.
//! Is is useful to write some, they are displayed both in the auto-generated docs with
//! `cargo doc --open` and by your IDE.

pub my_code_goes_here;
```